### PR TITLE
feat<client>: new ui

### PR DESCRIPTION
**Pull request by dip-land**
UI overhaul
added mobile UI

View live demo https://hp.dipped.dev/
When registering use code "1"

### CODE REVIEW

File: src/html/scripts/app.js [hunk 1]
```diff
@@ -258,10 +318,7 @@ document.getElementById("save_profile").onclick = ()=>{
...
+
 function moveChat(){
-    let temp = document.getElementById("messageContainer");
-    if((temp.scrollHeight - temp.clientHeight) <= (temp.scrollTop + 10)){
-        temp.scrollTop = temp.scrollHeight;
-    }
-}
\ No newline at end of file
+    messageContainer.scrollTo({top: messageContainer.scrollHeight});
+}
```
Comment (TenDRILLL) [inline src/html/scripts/app.js:338]: Wouldn't this change to moveChat scroll it ALWAYS? The idea is to replicate Discord's way, of not scrolling if you're scrolling up, but autoscroll if you're at the bottom.

Comment (dip-land) [reply src/html/scripts/app.js:338]: It does not scroll it always, only when you send something

Comment (TenDRILLL) [reply src/html/scripts/app.js:338]: Egro my previous point stands, it'd autoscroll on new message, completely discarding if you're scrolling the messages above or not.

Comment (dip-land) [reply src/html/scripts/app.js:338]: It scrolls only on your new message not other users new messages, unless I missed something and it does do that

**Pull request merged**